src/transpileSettings.js

```diff
@@ -0,0 +1,250 @@
+var fs = require('fs'),
+  path = require('path'),
+  shell = require('shelljs'),
+  Q = require('q'),
+  semver = require('semver');
+
+  var projectInfo = require(path.join(__dirname, 'monaca', 'projectInfo'));
+  var localPackageJSON = require(path.join(__dirname,'monaca', 'localPackageJSON'));
+
+  var getProjectInfoProperty = function(projectDir, property) {
+    return projectInfo.get(projectDir, property);
+  };
+
+  var setProjectInfoProperty = function(projectDir, property, value) {
+    return projectInfo.set(projectDir, property, value);
+  };
+
+  var getLocalPackageJSONProperty = function(projectDir, property) {
+    return localPackageJSON.get(projectDir, property);
+  };
+
+  var setLocalPackageJSONProperty = function(projectDir, property, value) {
+    return localPackageJSON.set(projectDir, property, value);
+  };
+
+  /**
+   * @method
+   * @class getMonacaPackageJSON
+   * @description
+   * Get monaca-lib package.json
+   * @return {Promise}
+   */
+  var getMonacaPackageJSON = function() {
+    var deferred = Q.defer();
+    try {
+      var monacaJsonFile = path.resolve(path.join(__dirname, '..', 'package.json')),
+        monacaDependencies = require(monacaJsonFile);
```

Comment:
this is the content of the whole `package.json` file, not `monacaDependencies`. Rename the variable.